src/map-implementation/__test-utils__.ts

```diff
@@ -80,7 +80,7 @@ export function getTestData(input: Partial<TestData> = {}): TestData {
     procedureName,
     accessibilityRequirements: failedRequirements,
   };
-  const outcomes = [expected];
+  const outcomes = input.outcomes ?? [expected];
```

Comment:
Thought it was needed, turn out it's unrelated but probably still the intention.